src/platform/forms/tests/address.unit.spec.js

```diff
@@ -7,9 +7,9 @@ import { expect } from 'chai';
 const domestic = {
   type: 'DOMESTIC',
   countryName: 'USA',
-  addressOne: '140 Rock Creek Church Rd NW',
-  addressTwo: 'Apt 57',
-  addressThree: 'Area Name',
+  addressLine1: '140 Rock Creek Church Rd NW',
```

Comment:
I couldn't find a reason to use the pre-existing naming for the address properties. Why not use what we have in redux? I updated the addressFormatter helper function accordingly.